CBattle/package/cog.py

```diff
@@ -90,6 +90,14 @@ async def page():
 
     @app_commands.command()
     async def add(self, interaction: discord.Interaction["BallsDexBot"], countryball: BallInstanceTransform):
+        """
+        Adds a ball from the battle..
+
+        Parameters
+        ----------
+        ball: countryball
```

Comment:
```suggestion
        countryball: BallInstance
```
`countryball` is not a valid Model. `ball` should be changed to `countryball` for translation.